.gitignore

```diff
@@ -127,6 +127,3 @@ dmypy.json
 
 # Pycharm project settings
 .idea/
-
-# Model specification files
-src/vivarium_census_prl_synth_pop/model_specifications/*.yaml
```

Comment:
I'm unsure if I should add this back once the new yaml gets merged. I'm assuming it's here just so that we're all free to have our own custom yaml files or something?